### DISCUSSION TITLE
Rpi hat V1 and openplotter V3, how to setup N2K ?

### DISCUSSION THREAD

**Discussion by LeifKlasson** [hatlabs/discussions#53] [Q&A]
Hi, I have a Rpi 4 with openplotter V3 and Rpi hat V1. I have run the setup script, and can see first step of N2K in Openplotter. But I can't edit this one, and no messages from N2K are seen in Openplotter?
![Capture2](https://github.com/hatlabs/discussions/assets/17064980/9e99130a-b555-48c8-ac84-c1653cdcad95)
![Capture](https://github.com/hatlabs/discussions/assets/17064980/6dfe61dd-9a78-47e0-b264-24afe795d0a5)

Should I installed it manually?
/Leif

**Comment by LeifKlasson** [comment]
Hi, soon a new season on water, and no solution to this yet ;-) ?

**Reply by mairas** [reply]
Hi, once you have installed the SH-RPi daemon by running the installer, the CAN interface is already configured. Do not use the OpenPlotter tools but instead, set up the NMEA 2000 Data Connection in Signal K server. Some older instructions can be found here: https://docs.hatlabs.fi/sh-rpi/docs/tutorials/openplotter-server/#configuring-signal-k-to-receive-nmea-2000-traffic
